package.json

```diff
@@ -29,6 +29,7 @@
     "isomorphic-unfetch": "^3.0.0",
     "js-cookie": "^2.2.1",
     "lodash": "^4.17.15",
+    "markdown-it": "^10.0.0",
```

Comment:
Wondering if we could use this instead of `markdown-to-jsx`... they seem a bit different, just a thought if we could consolidate or if we need to carry both. What do you think?

Reply:
from I gathered one really renders a component and the other just spits out a string of markdown

Reply:
Makes sense. I know you've worked closely with both so was curious your thoughts.